PRETTY_GOOD_PRACTICES.md

```diff
@@ -6,7 +6,7 @@
 # Pretty Good Practices
 
 Software development is complicated and often requires significant flexibility in terms of the patterns used.
-But...
+But,
 Using that flexibility in all cases results in difficult to read and buggy code.
```

Comment:
This was intentionally informal.  But, (`:]`) if we're changing it...  This would become a single line with the next to retain semantic newlines and should also drop the capital `U` to lowercase.